### DISCUSSION TITLE
Config: unsupported database driver postgres, using sqlite

### DISCUSSION THREAD

**Discussion by Olili2017** [photoprism/photoprism#2240] [Q&A] [answered]
I was able to deploy photoprism on a docker swarm. but when my machine went down, the prism server didn't come back up.

READING ERROR LOGS: kept on displaying the error below.

```
ph_photoprism.1.s8aymm5egk2v@ubuntu | /opt/photoprism/bin/photoprism start
ph_photoprism.1.s8aymm5egk2v@ubuntu | time="2022-04-11T14:43:40Z" level=info msg="PhotoPrism® needs your support!"
ph_photoprism.1.s8aymm5egk2v@ubuntu | time="2022-04-11T14:43:40Z" level=info msg="Visit https://docs.photoprism.app/funding/ to learn more."
ph_photoprism.1.s8aymm5egk2v@ubuntu | time="2022-04-11T14:43:40Z" level=warning msg="config: unsupported database driver postgres, using sqlite"
ph_photoprism.1.s8aymm5egk2v@ubuntu | panic: migration failed
ph_photoprism.1.s8aymm5egk2v@ubuntu |

[PhotoPrism - Funding](https://docs.photoprism.app/funding/)
User Guide
```

NOTE: I had set `PHOTOPRISM_DATABASE_DRIVER: postgres`.
And this had no problem when testing through deploying on my local machine.

Does anyone know why this is happening?

**Comment by lastzero** [comment, marked as answer]
See:
- https://github.com/photoprism/photoprism/issues/47

**Reply by Olili2017** [reply]
Got it. Thank you.

TL;DR; `If anyone needs to know that summary, @lastzero has it jotted down for us in the link below.`

https://github.com/photoprism/photoprism/issues/47#issuecomment-1016443349

The longer answer is in the link above in the main thread.
